scripts/run_benches_for_runtime.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Runs all benchmarks for all pallets, for a given runtime, provided by $1
+# Should be run on a reference machine to gain accurate benchmarks
+# current reference machine: https://github.com/paritytech/substrate/pull/5848
+
+runtime="$1"
+standard_args="--release --locked --features=runtime-benchmarks"
```

Comment:
we can add `--execution=wasm --wasm-execution=compiled` here

Reply:
`--chain "${runtime}-dev"` as well

Reply:
My bad, the 2 comments above would be a second variable.

Reply:
So the first variable is `cargo_args` and the second variable could be `benchmark_args`